controllers/clusteroperator.go

```diff
@@ -87,8 +90,7 @@ func (r *ProvisioningReconciler) createClusterOperator() (*osconfigv1.ClusterOpe
 // getOrCreateClusterOperator gets the existing CO, failing which it creates a new CO.
 func (r *ProvisioningReconciler) getOrCreateClusterOperator() (*osconfigv1.ClusterOperator, error) {
 	existing, err := r.OSClient.ConfigV1().ClusterOperators().Get(context.Background(), clusterOperatorName, metav1.GetOptions{})
-	if errors.IsNotFound(err) {
-		r.Log.V(1).Info("cluster baremetal operator does not exist, creating a new one.")
```

Comment:
This msg is useful to figure out if an existing CO was used or a new one was created. I agree with the general idea of not logging and the returning error. But, if the a log message indicates an intermediate state that then caused action1 or action2 to be taken, then I think it would be helpful to log it at that time before moving on.

Reply:
As things are now, we get 2 log messages when we create a CO. This removes the one that anticipates creating it, in favor of the success message that comes in createClusterOperator() on line 84.

Reply:
Yes, either one of these msgs should do. And I agree that the msg after creating one is better than the one anticipating creating of one.